modules/swagger-codegen/src/main/resources/typescript-angular2/api.mustache

```diff
@@ -205,9 +209,13 @@ export class {{classname}} {
             body: formParams.toString(),
 {{/hasFormParams}}
             search: queryParameters,
-            withCredentials:true
+            withCredentials:isWithCredentials
```

Comment:
In the case of the `withCredentials` is not defined in the configuration object, I just define a local var to set the default value to `false`.